src/handler/handlerargstest.cpp

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2025 Fastly, Inc.
+ *
+ * This file is part of Pushpin.
+ *
+ * $FANOUT_BEGIN_LICENSE:APACHE2$
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * $FANOUT_END_LICENSE$
+ */
+
+#include "config.h"
+#include "handlerapp.h"
+#include "settings.h"
+#include "test.h"
+#include "handlerargsdata.h"
+#include "rust/bindings.h"
+#include "log.h"
+
+void handlerargstest()
+{
+
+    // Create dummy argc/argv for QCoreApplication
+    int argc = 1;
+    char appName[] = "pushpin-handler";
+    char* argv[] = { appName, nullptr };
+
+    QCoreApplication qapp(argc, argv);
```

Comment:
we are trying to avoid `QCoreApplication` now in tests. I wonder if this can be removed and it still works?